src/hooks/chatSidebar.ts

```diff
@@ -83,3 +89,24 @@ export const useChatSidebarInfo = () => {
     venueChatTabTitle: `${chatTitle} Chat`,
   };
 };
+
+export const useChatSidebarPinned = () => {
+  const dispatch = useDispatch();
+  const { width } = useWindowSize();
+  const venue = useSelector(currentVenueSelectorData);
+  const isPinned = useMemo(
+    () =>
+      width > LARGE_SCREEN_WIDTH && venue?.template !== VenueTemplate.audience,
+    [width, venue]
+  );
+
+  useEffect(() => {
+    if (isPinned) {
+      dispatch(setChatSidebarVisibility(true));
+    }
+  }, [dispatch, isPinned, width]);
```

Comment:
I don't think you want to be including `width` in the dependencies here?:

```suggestion
  useEffect(() => {
    if (isPinned) {
      dispatch(setChatSidebarVisibility(true));
    }
  }, [dispatch, isPinned]);
```